### PR TITLE
feat: implement Datasets.Blob in K8 target

### LINKED ISSUE #453: feat: implement Datasets.Blob in non-local targets

### PR DESCRIPTION
**Pull request by aavarghese**
Fixes #453

### CODE REVIEW

File: pkg/be/kubernetes/shell/chart/templates/blob.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{{- if .Values.blob.cm.data }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ print (.Release.Name | trunc 44) "-blob" }}
```
Comment (starpit) [inline pkg/be/kubernetes/shell/chart/templates/blob.yaml:5]: i think the idea of `trunc 44` is that it should be 53 minus len("-blob") so `trunc 48` i guess?

**Review by starpit — CHANGES_REQUESTED** [review]
curious: could we add the blob to the existing workdir.cm.data instead of creating a new configmap for the blob data?

**Comment by aavarghese** [comment]
> curious: could we add the blob to the existing workdir.cm.data instead of creating a new configmap for the blob data?

@starpit previously, I had the blob changes together with the other dataset code. In order to use the existing CM, I now have the blob data operations along with the codespec stuff. wdyt?

**Comment by starpit** [comment]
> > curious: could we add the blob to the existing workdir.cm.data instead of creating a new configmap for the blob data?
> 
> @starpit previously, I had the blob changes together with the other dataset code. In order to use the existing CM, I now have the blob data operations along with the codespec stuff. wdyt?

thanks!

**Review by starpit — APPROVED** [review]
lgtm

**Pull request merged**